requirements.txt

```diff
@@ -1,15 +1,20 @@
+aniso8601==8.1.0
+autopep8==1.5.4
 certifi==2020.6.20
 chardet==3.0.4
-idna==2.10
-requests==2.24.1
-urllib3==1.25.10
 click==7.1.2
 Flask==1.1.2
+Flask-RESTful==0.3.5
 Flask-SQLAlchemy==2.4.4
+idna==2.10
 itsdangerous==1.1.0
 Jinja2==2.11.2
 MarkupSafe==1.1.1
+pycodestyle==2.6.0
+pytz==2020.4
+requests==2.25.0
+six==1.15.0
 SQLAlchemy==1.3.20
+toml==0.10.2
+urllib3==1.25.10
 Werkzeug==1.0.1
-Flask-RESTful==0.3.5
```

Comment:
some requiriements were missgin here eg FLASK. one "safe" way to store all ur requirements is to run `flask freeze` and copy the outcome list and put it here.